examples/experimental/dagster-airlift/dagster_airlift/core/airflow_instance.py

```diff
@@ -337,6 +338,19 @@ def details_url(self) -> str:
     def log_url(self) -> str:
         return f"{self.details_url}&tab=logs"
 
+    @property
+    def logical_date(self) -> datetime.datetime:
+        """Returns the airflow-coined "logical date" from the task instance metadata.
+        The logical date refers to the starting time of the "data interval" that the overall dag run is processing.
+        In airflow < 2.2, this was set as the execution_date parameter in the task instance metadata.
+        """
+        # In airflow < 2.2, execution_date is set instead of logical_date.
+        logical_date_str = check.not_none(
+            self.metadata.get("logical_date") or self.metadata.get("execution_date"),
+            "Expected one of execution_date or logical_date to be set.",
+        )
```

Comment:
I don't think we should hard error on this. I can imagine cases where folks have lightly forked airflow (or jammed in values. It's Python you can do anything). 

So instead I think we should let `logical_data` take precedence, but not hard error.

Reply:
Ah I misread the code from the comment. I thought you were hard erroring if they were both set.